apps/web/src/app/booking/page.tsx

```
@@ -0,0 +1,181 @@
+    guests: number;
+    totalAmount: number;
+  }) => {
+    if (!isConnected) {
+      setShowWalletModal(true);
+      return;
+    }
+
+    try {
+      // Process payment using Stellar
+      const transactionHash = await processPayment(
+        publicKey!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)